docs/content/installation/integrations/app-protect-waf-v5/installation.md

```diff
@@ -0,0 +1,241 @@
+
+{{<bootstrap-table "table table-striped table-bordered">}}
+| Makefile Target           | Description                                                       | Compatible Systems  |
+|---------------------------|-------------------------------------------------------------------|---------------------|
+| **alpine-image-nap-v5-plus-fips** | Builds a Alpine-based image with NGINX Plus and the [NGINX App Protect WAF v5](/nginx-app-protect-waf-v5/) module with FIPS. | Alpine  |
+| **debian-image-nap-v5-plus** | Builds a Debian-based image with NGINX Plus and the [NGINX App Protect WAF v5](/nginx-app-protect-waf-v5/) module. | Debian  |
+| **ubi-image-nap-v5-plus**    | Builds a UBI-based image with NGINX Plus and the [NGINX App Protect WAF v5](/nginx-app-protect-waf-v5/) module. | OpenShift |
+| **ubi-image-nap-dos-v5-plus** | Builds a UBI-based image with NGINX Plus, [NGINX App Protect WAF v5](/nginx-app-protect-waf-v5/), and [NGINX App Protect DoS](/nginx-app-protect-dos/). | OpenShift |
+{{</bootstrap-table>}}
+
+<br>
+
+{{<see-also>}} For the complete list of _Makefile_ targets and customizable variables, see the [Building NGINX Ingress Controller]({{< relref "installation/building-nginx-ingress-controller.md#makefile-details" >}}) guide. {{</see-also>}}
+
+If you intend to use [external references](/nginx-app-protect-waf/v5/configuration/#external-references) in NGINX App Protect WAF policies, you may want to provide a custom CA certificate to authenticate with the hosting server.
+
+To do so, place the `*.crt` file in the build folder and uncomment the lines following this comment:
+`#Uncomment the lines below if you want to install a custom CA certificate`
+
+{{<warning>}} External references are deprecated in NGINX Ingress Controller and will not be supported in future releases. {{</warning>}}
+
+---
+
+## Push the images to your private registry
+
+Once you've successfully pulled the WAF v5 manager and enforcer images and built the NGINX Ingress Controller image with NGINX App Protect WAF v5, the next step is to upload them to your private Docker registry. This makes the image available for deployment to your Kubernetes cluster.
+
+To upload the image, run the following command. If you're using a custom tag, add `TAG=your-tag` to the end of the command. Replace `<my-docker-registry>` with your private registry's path.
+
+```shell
+make push PREFIX=<my-docker-registry>/nginx-plus-ingress
+```
+
+To upload the WAF config manager and enforcer images run the following commands:
+
+```shell
+docker push <my-docker-registry>/waf-config-mgr:<your-tag>
+```
+
+```shell
+docker push <my-docker-registry>/waf-enforcer:<your-tag>
+```
+
+---
+
+## Set up role-based access control (RBAC) {#set-up-rbac}
+
+{{< include "rbac/set-up-rbac.md" >}}
+
+---
+
+
+{{< include "installation/create-custom-resources.md" >}}
+
+
+## Deploy NGINX Ingress Controller {#deploy-ingress-controller}
```

Comment:
We should absolutely add a section in here about deploying NIC with WAF v5 using Helm since we have so many different values that are now configurable with Helm